classes/class-dibs-easy-gateway.php

```diff
@@ -291,6 +293,25 @@ public function process_dibs_payment_in_order( $order_id ) {
 		}
 	}
 
+	/**
+	 * Save shipping reference to Order.
+	 *
+	 * @param int $order_id order id.
+	 * @return void
+	 */
+	public function nets_save_shipping_reference_to_order( $order_id ) {
```

Comment:
Is there a specific reason that the function name is called nets_save_shipping_reference_to_order? Since this is a function within a class it could just be named save_shipping_reference_to_order? Or is it good to have Nets as part of the reference here?

Reply:
This is a fair comment, if its within our class the nets prefix is not really needed.